certbot/main.py

```diff
@@ -651,34 +661,24 @@ def obtain_cert(config, plugins, lineage=None):
         raise
     le_client = _init_le_client(config, auth, installer)
 
-    # SHOWTIME: Possibly obtain/renew a cert, and set action to renew | newcert | reinstall
-    if config.csr is None: # the common case
-        domains, certname = _find_domains_or_certname(config, installer)
-        action, _ = _auth_from_available(le_client, config, domains, certname, lineage)
-    else:
-        assert lineage is None, "Did not expect a CSR with a RenewableCert"
+    if config.csr:
         _csr_obtain_cert(config, le_client)
-        action = "newcert"
+        _report_new_cert(config, print_fullchain_path=False)
+        _suggest_donation(config)
+        return
 
-    # POSTPRODUCTION: Cleanup, deployment & reporting
-    notify = zope.component.getUtility(interfaces.IDisplay).notification
-    if config.dry_run:
-        _report_successful_dry_run(config)
-    elif config.verb == "renew":
```

Comment:
With the logic in this `if` statement removed (and not put in `renew_cert`), we now no longer reload user's webservers upon renewal. If they used the Apache or Nginx plugin, their servers will continue to use the old certificates upon renewal.